app/server/tests/routes/apps.test.ts

```diff
@@ -1,27 +1,26 @@
 import { AppsController } from "../../src/controllers/appsController";
 import { SessionsController } from "../../src/controllers/sessionsController";
+import bodyParser from "body-parser";
 
-describe("odin routes", () => {
-    const express = require("express");
-    const bodyParser = require("body-parser");
+const { mockRouter } = vi.hoisted(() => ({
+    mockRouter: {
+        get: vi.fn(),
+        post: vi.fn()
+    }
+}));
 
-    const mockRouter = {
-        get: jest.fn(),
-        post: jest.fn()
-    };
-    const realRouter = express.Router;
+vi.mock("express", () => ({
+    Router: () => mockRouter
+}));
 
-    const spyText = jest.spyOn(bodyParser, "text");
-
-    beforeAll(() => {
-        express.Router = () => mockRouter;
+describe("odin routes", async () => {
+    beforeEach(() => {
+        vi.resetAllMocks()
```

Comment:
this was weird, we were importing and then reassigning an imported thing as something else and then resetting it with the real router? this is just a roundabout of just mocking the package so thats what i changed this to, now im just mocking the router

Reply:
have done this same thing in a couple other places below